packages/ecl-builder/scripts/copy.js

```diff
@@ -1,11 +1,20 @@
-const copy = require('fs-extra').copy;
+const path = require('path');
+const copy = require('ncp').ncp;
+const mkdirp = require('mkdirp');
 
 module.exports = (from, to) => {
-  copy(from, to, (err) => {
-    if (err) {
-      return console.error(err);
+  mkdirp(path.dirname(to), (mkdirpErr) => {
+    if (mkdirpErr) {
+      console.error(mkdirpErr);
+      process.exit(1);
     }
 
-    return 0;
+    copy(from, to, (err) => {
```

Comment:
As there's not much about handling errors here, just `copy(from, to);` works also.
I tried for example, naming the `fonts` folder `fontz` and if the error is not handled, the process doesn't stop and does not complain, simply does not copy the folder.
Otherwise, it's good!